ui/cypress/e2e/explorer.test.ts

```diff
@@ -623,7 +623,7 @@ describe('DataExplorer', () => {
         })
       })
 
-      it('can view table data', () => {
+      it('can view table data & sort values numerically', () => {
```

Comment:
this test fails when run with `it.only` which means it's coupled to another part of the test. i think this needs to get fixed.

Reply:
I'll take a look and try to reproduce it locally and see whether I can make it less flakey

Reply:
until we sort this out, i think you should `skip` this test and link to this issue: https://github.com/influxdata/influxdb/issues/16253